docs/products/tools/marketplace/guides/passbolt/index.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: Deploy Passbolt Through The Linode Marketplace
+title: "Deploy Passbolt Through The Linode Marketplace"
+description: 'Deploy Passbolt password manager through the Linode Marketplace.'
+og_description: 'Deploy Passbolt password manager through the Linode Marketplace.'
+keywords: ['passbolt','password manager','security','authentication']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+authors: ["Linode"]
+published: 2024-02-05
+modified_by:
+  name: Linode
+---
+
+[Passbolt Password Manager](https://github.com/passbolt/passbolt_api) is an open-source password manager designed for teams and businesses. It allows users to securely store, share and manage passwords.
+
+## Deploying a Marketplace App
+
+{{< content "deploy-marketplace-apps-shortguide">}}
+
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** Passbolt should be fully installed within 5-10 minutes after the Compute Instance has finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions:** Ubuntu 22.04 LTS
+- **Recommended plan:** We recommend a 4GB Dedicated CPU or Shared Compute instance for Passbolt.
```

Comment:
Per our legal department, we don't recommend anything to our customers. 

Replace "Recommended" with "Suggested"
Replace "We recommend" with "For the best results, use a 4GB Dedicated CPU or Shared Compute instance for Passbolt."